deps/icu-small/source/common/uresdata.h

```diff
@@ -399,7 +399,7 @@ typedef struct ResourceData {
     UBool useNativeStrcmp;
 } ResourceData;
 
-struct UResourceDataEntry;   // forward declared for ResoureDataValue below; actually defined in uresimp.h
```

Comment:
This fix would need to be made in the upstream dependency. This change should be removed from this PR.